uptrain/tests/test_algorithms_clustering.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+from uptrain.core.classes.algorithms.clustering import Clustering
```

Comment:
@a-r-r-o-w The test idea is great. However, it should not be this way. Right now, it's checking only the algorithm (which will most likely be correct). But we need to check the entire pipeline with UpTrain. The test should define UpTrain config and framework logging to check for clustering. The same goes for popularity bias.

Reply:
I see, thanks. I will look into how a synthetic test could be created here and open a new PR when I have a little better understanding of how I could use the pipeline for the test.